jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -427,16 +472,17 @@ then
       do
         if [ -r "$XMLFILE" ] && [ -f "$XMLFILE" ]
         then
-          JETTY_ARGS=(${JETTY_ARGS[*]} "$XMLFILE")
+          JETTY_ARGS[${#JETTY_ARGS[@]}]=$XMLFILE
```

Comment:
I think we still need the double quotes
```suggestion
          JETTY_ARGS[${#JETTY_ARGS[@]}]="$XMLFILE"
```

I did the following test:
```bash
[] TEST=("foo" "bar")

[] echo ${FOO[*]}
foo bar bob

[] TEST=("foo" "bar")

[] echo ${TEST[*]}
foo bar

[] TEST[2]=bob

[] echo ${TEST[*]}
foo bar bob

[] TEST[3]=with space
bash: `TEST[3]': not a valid identifier
Command 'space' not found, but can be installed with:
sudo snap install space

[] TEST[3]="with space"

[] echo ${TEST[*]}
foo bar bob with space

```

Reply:
Oh!!!! It works because it is a single variable:
```sh
[] WITH_SPACES='var with spaces'

[] TEST[4]=$WITH_SPACES
```

This is sooooo subtle!  bash is getting more perl like!